packages/document-service-api/src/lib/blobStorage.js

```diff
@@ -115,8 +115,23 @@ const uploadToBlobStorage = (blobStorageClient) => {
     );
 };
 
+const deleteFromBlobStorage = async (location) => {
+  connectToBlobStorage();
+
+  const doc = containerClient.getBlockBlobClient(location);
```

Comment:
Should this be await? Probably not, I can't see getBlockBlobClient to check.

Reply:
https://docs.microsoft.com/en-us/javascript/api/@azure/storage-blob/containerclient?view=azure-node-latest#getBlockBlobClient_string_

<img width="1005" alt="Screenshot 2021-04-12 at 09 03 04" src="https://user-images.githubusercontent.com/6424029/114361319-e6bfc480-9b6d-11eb-9ef2-ca30304e6358.png">

seems its ok